packages/contracts-bedrock/contracts/test/RLP.t.sol

```diff
@@ -10,7 +10,7 @@ import { Bytes32AddressLib } from "@rari-capital/solmate/src/utils/Bytes32Addres
 library LibRLP {
     using Bytes32AddressLib for bytes32;
 
-    function computeAddress(address deployer, uint256 nonce) public pure returns (address) {
+    function computeAddress(address deployer, uint256 nonce) internal pure returns (address) {
```

Comment:
This is in a test file. Why is it an issue for state surgery?

Reply:
Its an issue because artifacts are created for it